data/benefits/javascript/caf-val-de-marne-aide-bafa-formation-generale.yml

```diff
@@ -29,4 +28,7 @@ conditions_generales:
   - type: age
     operator: <=
     value: 25
+  - type: regime_securite_sociale
+    not: true
+    value: regime_agricole
```

Comment:
Ici j'aurais plutôt utilisé un autre terme que "not" et "value". Je n'arrive pas à trouver un mot adapté, mais on pourrait imaginer quelque chose du genre : 
```yaml
conditions_generales:
- type: regime_securite_sociale
  exclude: regime_agricole
```

L'avantage c'est que ça permettrait de n'avoir qu'une instruction au lieu de deux et ça serait (de mon point de vue) plus clair

Reply:
Tu aurais créer un champ include et un champ exclude ou deux critères séparés ?

Reply:
J'ai gardé values pour conserver la logique utilisée jusqu'a présent

Reply:
@guillett ton avis, moi je suis partagé

Reply:
Pour conserver ce qu'il s'est dit
![image](https://user-images.githubusercontent.com/65901733/145982487-33194ea5-73a4-4d7f-884e-8e3763b2ab5c.png)